apps/web-mzima-client/src/env.json

```diff
@@ -1,6 +1,6 @@
 {
   "production": true,
-  "backend_url": "http://localhost:8080/",
+  "backend_url": "https://mzima-dev-api.staging.ush.zone/",
```

Comment:
I think I have seen this line changing back and forth in a couple PRs , maybe we should just .gitignore it 🤷